RELEASES.md

```diff
@@ -1,3 +1,120 @@
+Version 1.50.0 (2021-02-11)
+============================
+
+Language
+-----------------------
+- [You can now use `const` values for `T` in `[T; N]` array expressions.][79270]
```

Comment:
`T` is normally used as a placeholder for a type, but here it's being used as a placeholder for a value. Can I suggest `x` instead?